a10_octavia/controller/healthmanager/a10_health_manager.py

```diff
@@ -84,12 +84,15 @@ def health_check(self):
             'failover_cancelled': 0,
         }
         futs = []
-        
+        failover_wait_time = CONF.health_manager.heartbeat_timeout
```

Comment:
Make sure to update this to `a10_health_manager` after the config changes have been made. Might be best if you just do it now.

Reply:
I will not do it for this PR, but will make changes in further PR, or it might break something.